tools/dotnet-linker/LinkerConfiguration.cs

```diff
@@ -47,7 +47,8 @@ public class LinkerConfiguration {
 
 		// The list of assemblies is populated in CollectAssembliesStep.
 		public List<AssemblyDefinition> Assemblies = new List<AssemblyDefinition> ();
-		
+		public Dictionary<string, AssemblyDefinition> AssembliesByName = new Dictionary<string, AssemblyDefinition> ();
```

Comment:
Are assembly names case-sensitive?

Reply:
They're not, good catch (removing this logic anyway)